packages/field-base/src/helper-text-mixin.d.ts

```diff
@@ -18,7 +18,7 @@ interface HelperTextMixin extends SlotMixin {
   /**
    * String used for the helper text.
    */
-  helperText: string;
+  helperText: string | null | undefined;
```

Comment:
We probably should update `vaadin-select` to remove helper text from there:

https://github.com/vaadin/web-components/blob/285be3011b894f67613875a3615f35cb3ba5809b/packages/select/src/vaadin-select.d.ts#L142-L145

Reply:
Right, fixed.